test/services/cert/rhsm_client_test.rb

```diff
@@ -5,36 +5,48 @@
-      /gkhjiyicxw+uxRHuNyMClM8Q5WVj4CpxrPHwZvN1OhM8D3VDnEaZj6J2k5g55Fl
-      3qQXsk8lPEJ2I5D00Up2cpDBy+CXj5zm/shmqEJlGOxILjpCzNhqER/YdBlGPP9b
-      okjCBjwYlp5cNyAJSQscLF7rj/iOJYhRdetWMZg=
-      -----END CERTIFICATE-----'.freeze
+    -----BEGIN CERTIFICATE-----
+    MIIGjDCCBHSgAwIBAgIIabvFmF/tIwkwDQYJKoZIhvcNAQELBQAwgZcxCzAJBgNV
+    BAYTAlVTMRcwFQYDVQQIDA5Ob3J0aCBDYXJvbGluYTEQMA4GA1UEBwwHUmFsZWln
+    aDEQMA4GA1UECgwHS2F0ZWxsbzEUMBIGA1UECwwLU29tZU9yZ1VuaXQxNTAzBgNV
+    BAMMLGNlbnRvczcta2F0ZWxsby1jYW5kbGVwaW4uYXJlYTUyLmV4YW1wbGUuY29t
+    MB4XDTIyMDEyMDIwMTIwNVoXDTM4MDEyMDIxMTIwNVowTjEdMBsGA1UECgwURGVm
+    YXVsdF9Pcmdhbml6YXRpb24xLTArBgNVBAMMJDU3YjI2Y2UzLWU0ZmUtNGY3OS1h
+    Mzc3LTJjNGQ5ZGVhNTIwNDCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIB
+    ALikZKO/50yG6qcWkSIr6IzPT8ohENEro6TERhWMCAuwikHE+MKdhPpyUIOkdt56
+    GXRU2zGhuxP4o+uG4B6r1EtG6Apvy5J6asL+pXHIUEvLG4HQmNQp8AqztAmS/Fh+
+    5j6n2PyD7/iF5Moozf7Ew+HJJvjy+RPHin3EEm3QvWcG65/5HGQHF2F9vUfo2kAE
+    ENhum2RLJ0EYZqBz7FkkBzS+onkY/uwmuRkxm0v4sHdQk1Fe5tLnBBm6pFDZzswB
+    v134Gi/j6I865i9FvvPvxrOE+MsBW/x1JAwj54IMa2HDINQmAY8jWWyiPGUY5yLl
+    kwnJ/CUBs+6qqKoUt+8H4TpC9dHStEdf2u1+nm+MgjRLm07piWo/Qp/bDh1OwfBT
+    bsRl2riE/wSofjex6X8ZjRxImhWNjxvzJAn/VD6p5+rpxFDu5RjCoKsGF9Tbn5Vr
+    /QRsfuiy9kwrsQYfp3tBAgyKsblTUncCT4KMTtVDbIm8MxlTLIDCjPRxLwhB1yan
+    05FuYHMHAMKkH6NCypuGA4YF1XSPMgO6SfBLz61yoAqMEXT9ugkj7gFt8bFzMmVn
+    b2BXNBXnC13oWG+EPd3THTeFEIZpbljZvGciqXTCKBnWD3tNcQx+ZQEbmhfNiwMg
+    hVv6lonLbPat4zK3JkW920/Vc+rfdoPRbxImsjMKh24BAgMBAAGjggEiMIIBHjAO
+    BgNVHQ8BAf8EBAMCBLAwEwYDVR0lBAwwCgYIKwYBBQUHAwIwCQYDVR0TBAIwADAR
+    BglghkgBhvhCAQEEBAMCBaAwHQYDVR0OBBYEFOx+heEao9Xx+vY/e3ozUT2FeM3i
+    MB8GA1UdIwQYMBaAFBnHeIYJrfaf8/3O44D+GvLA2Pc3MIGYBgNVHREEgZAwgY2k
+    UDBOMR0wGwYDVQQKDBREZWZhdWx0X09yZ2FuaXphdGlvbjEtMCsGA1UEAwwkNTdi
+    MjZjZTMtZTRmZS00Zjc5LWEzNzctMmM0ZDlkZWE1MjA0pDkwNzE1MDMGA1UEAwws
+    Y2VudG9zNy1rYXRlbGxvLWNhbmRsZXBpbi5hcmVhNTIuZXhhbXBsZS5jb20wDQYJ
+    KoZIhvcNAQELBQADggIBAAirOKFelb8FqTa6aaFb7JA7A3bQw7gjCDgaLiWF2KrT
+    RUa1ayHQJ+Tm5jJPo9RNLzDnpKNjVFDK/E9zB8tcOTg8dZoeCvQ6FvjbRTi/8MWP
+    Be5m3ef64a8tQxhdJPeMkRAjKnUzsGZbBjttZue3B89XisKytssk1tkXnY62V42Z
+    m5aAiUfkTr/jeZkuZyLI9uF7Np4rws44/XmoEJp/LNWKmCTtvgIR6NP1W+J2LA/g
+    LOIDLNTyy8Ju95ZDdNQ8KJC+Hrb+OiL86SvwHK9gTAqNqbKmFmQkAtpsgmcObRZF
+    tQRzaHPwR+Pj1wmIoi6UMbeUKuPbVYRf2qVPtvtqFAIGhVBobmIHSE1+G2as0zqZ
+    4LkssrjsyRR02K2aoIOl6IB0s1U2vZ8SRUI4HabSYX/rw9iEdWO1f/gD76YBxSxF
+    5d/rtZUAUQEgS1hdrPlzeNsG5H8Mf05lJMghH8GFJfjkqm6OHkGYcJGCns1Mpai0
+    4RS8QEA83rtEmSdY/oubjNGUMeXk0mjkOrSIblSUYixJJYdi+66IjePyCGMRhtd4
+    0TV5WB3ONylUv777yuHRICFZyny63C1vtUIZMGtu67qYjKk4uMKnHC5lWMaSmepG
+    ifSe+u12fpxjD0UirYSMj38vDtCbcFSPcAAu/1YKw4uvN5zcv+YI91tjOkMJuVah
+    -----END CERTIFICATE-----'.freeze
 
     def test_uuid
       rhsm_cert = ::Cert::RhsmClient.new(CERT)
-      assert_equal rhsm_cert.uuid, '14e98155-731a-4cae-b151-5c504cc30e1a'
+      refute_equal rhsm_cert.uuid, rhsm_cert.cert.subject.to_s
```

Comment:
If you've already asserted what it should be equal to I wonder what value another assertion that states it should _not_ equal something else adds

Reply:
Fair point will remove the other

Reply:
I think it can add a lot of value to a test to assert both that something _is_ one way and _is not_ another way! It of course doesn't make any difference to the test's functionality, but I feel like tests' secondary purpose is being read by humans, and being explicit in this way can be a great way to project your intentions.

Reply:
I suppose it could be a comment instead of an actual assertion, if that's more palatable :yum:

Reply:
That is what I was going for, to make sure we don't see the org in there, but I also see the other side where we are just grabbing the CN